src/Spryker/Zed/EventBehavior/Business/Model/EventResourceQueryContainerManager.php

```diff
@@ -118,7 +118,9 @@ protected function createEventResourceQueryContainerPluginIterator(EventResource
     protected function triggerBulk(EventResourceQueryContainerPluginInterface $plugin, array $ids): void
     {
         $eventEntityTransfers = array_map(function ($id) {
-            return (new EventEntityTransfer())->setId($id);
+            return (new EventEntityTransfer())
+                ->setId($id)
+                ->setTimestamp(time());
```

Comment:
so we just set time from now, but we need to compare it with the time from the message. This is not according AC

Reply:
are those changes mentioned in the changelogs?

Reply:
yes, it's enough here because we publish specific entities, we don't modify them.